docs/chunking.md

```diff
@@ -0,0 +1,107 @@
+# Chunk text with SQL functions
+
+The `ai.chunk_text` and `ai.chunk_text_recursively` functions allow you to split text into smaller chunks.
+
+## Example usage
+
+Given a table like this
+
+```sql
+create table blog
+( id int not null primary key
+, title text
+, body text
+);
+```
+
+You can chunk the text in the `body` column like this
+
+```sql
+select
+  b.id
+, b.title
+, c.seq
+, c.chunk
+from blog b
+cross join lateral ai.chunk_text(b.body) c
+order by b.id, c.seq
+;
+```
+
+## chunk_text
+
+Splits text into chunks using a separator
+
+| Name                | Type  | Default | Required | Description                                               |
+|---------------------|-------|---------|----------|-----------------------------------------------------------|
+| input               | text  | -       | ✔        | The text to split into chunks                             |
+| chunk_size          | int   | -       | ✖        | The target size of a chunk in characters                  |
+| chunk_overlap       | int   | -       | ✖        | The target amount of overlapping characters in each chunk |
+| separator           | text  | -       | ✖        | The text to split on                                      |
```

Comment:
I find this somewhat confusing. Under the hood we're using LangChain's CharacterTextSplitter, so while _we_ don't set a default value in the `chunk_text` function, the defaults for these are `chunk_size = 4000`, `chunk_overlap = 200`, and `separator = '\n\n'`. I would suggest that we surface both the fact that we're using CharacterTextSplitter, and what the actual defaults are.

Reply:
done